doc/playbook.rst

```diff
@@ -471,38 +471,41 @@ For passwordless login, create a ``~/.pgpass`` file with contents:
 
 .. code-block::
 
+   # Dev APDBs
    usdf-prompt-processing-dev.slac.stanford.edu:5432:lsst-devl:rubin:PASSWORD
-   usdf-prompt-processing-dev.slac.stanford.edu:5432:ppcentralbutler:latiss_prompt:PASSWORD
-   usdf-prompt-processing-dev.slac.stanford.edu:5432:ppcentralbutler:hsc_prompt:PASSWORD
-   usdf-prompt-processing-dev.slac.stanford.edu:5432:ppcentralbutler:lsstcomcamsim_prompt:PASSWORD
+   # Dev central repo, can also go in db-auth (see below)
+   usdf-prompt-processing-dev.slac.stanford.edu:5432:ppcentralbutler:pp:PASSWORD
 
 and execute ``chmod 0600 ~/.pgpass``.
 
 Cassandra
 ^^^^^^^^^
 
-We have a Cassandra cluster at the USDF on dedicated hardware, that is currently deployed in parallell across 12 nodes.
+We have a Cassandra cluster at the USDF on dedicated hardware, that is currently deployed in parallel across 12 nodes.
 Of those, 6 are reserved for Andy Salnikov's development and testing, and 6 are available for Prompt Processing.
-The status of the cluster can be monitored with the `cassandra_dashboard`_ in grafana.
-
-.. _cassandra_dashboard: https://grafana.slac.stanford.edu/d/d7d52e6b-e376-49dc-8ef8-e4742dd229a9/cassandra-system-metrics?orgId=1&refresh=5m
-
 The nodes available for Prompt Processing are ``sdfk8sk001`` through ``sdfk8sk006``.
 
 To access the Cassandra cluster, you must add credentials to your ``~/.lsst/db-auth.yaml``.
-The appropriate credentials are stored in the SLAC Vault, under ``rubin/usdf-apdb-dev/cassandra``.
+The appropriate credentials are stored in the `SLAC Vault <https://vault.slac.stanford.edu/ui/vault/secrets/secret/show/rubin/usdf-apdb-dev/cassandra>`_.
 Add the following to your ``db-auth.yaml``, replacing ``PORT`` and ``PASSWORD`` from the Vault:
 
 .. code-block:: sh
 
-   - url: cassandra://sdfk8sk001.sdf.slac.stanford.edu:PORT/*
+   # Cassandra dev APDBs
+   - url: cassandra://sdfk8sk001.sdf.slac.stanford.edu:PORT/pp_apdb_*_dev
      username: apdb
      password: PASSWORD
+   # Dev central repo, can also go in .pgpass (see above)
+   - url: postgresql://usdf-prompt-processing-dev.slac.stanford.edu/ppcentralbutler
+     username: pp
+     password: PASSWORD
+   # Workaround for list-cassandra not having keyspace-agnostic credentials, MUST go after all other entries
+   - url: cassandra://sdfk8sk001.sdf.slac.stanford.edu:9042/*
+     username: ANY_CASSANDRA_ACCOUNT
```

Comment:
Do you mean `ANY_CASSANDRA_ACCOUNT` literally here and not `apdb`?

Reply:
You can use `apdb`, `apdb-prod`, or anything else, so long as it's a valid account. How to express that?